src/components/redirects/redirects-routes.ts

```diff
@@ -21,11 +21,11 @@ router.get(PATH_DATA.THANKS_TXT.url, (_, res) => (
   )
 );
 
-async function redirectManageYourAccount(
+async function redirectToSecurity(
```

Comment:
I think this is triggering some sonarcloud warnings.
Does this need to be an async function?

If so, there may be a conflict, sonarcloud thinks the datatype should be `void` and not `Promise<void>`

Reply:
I don't _think_ it needs to be async so I changed to a regular one. The weird thing is I didn't do anything about the other Sonarcloud warning, yet it disappeared.